libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -153,6 +163,9 @@ void AP_Camera::trigger_pic()
     case AP_CAMERA_TRIGGER_TYPE_RELAY:
         relay_pic();                    // basic relay activation
         break;
+    case AP_CAMERA_TRIGGER_TYPE_GOPRO:
```

Comment:
What's missing I think is supporting the mavlink messages to start/stop video? That's the piece that people would really like for runcam

Reply:
Here we just made the support for gopro messaging not the full support for camera api. It would come on another PR